examples/azure-operator-lab-chart/templates/configmap.yaml

```diff
@@ -10,9 +10,6 @@ data:
         clientsecret: "{{.Values.azure.clientSecret}}"
         subscriptionid: "{{.Values.azure.subscriptionId}}"
         tenantid: "{{.Values.azure.tenantId}}"
-        template:
-          uri:
-            version: "{{.Values.azure.templateUriVersion}}"
```

Comment:
Lets make sure the installations repo is also cleaned up.

Reply:
I don't think we have anything in the installations repo. It's not deployed anywhere. Only local testing.